src/store/index.js

```diff
@@ -1,12 +1,24 @@
+import { persistStore, persistReducer } from "redux-persist";
+import storage from "redux-persist/lib/storage"; // defaults to localStorage for web
```

Comment:
🚀

Reply:
:). I got to know about this library from David.